pootle/templates/browser/index.html

```diff
@@ -10,68 +10,105 @@
-              {% endif %}
-            </td>
-            <td class="stats-percentage"><span></span>%</td>
-          </tr>
-          {% endfor %}
-        </tbody>
-      </table>
-    </div>
-    <div class="bd">
-      <table class="last">
-        <tbody>
-          <tr id="js-last-updated">
-            <td>{% trans "Last updated" %}</td>
-            <td class="last-updated"></td>
-          </tr>
-          <tr id="js-last-action">
-            <td>{% trans "Last action" %}</td>
-            <td class="last-action"></td>
-          </tr>
-        </tbody>
-      </table>
+    <a id="js-path-summary" class="expand-stats" href="#">
+      <i id="js-expand-icon" class="icon-expand-stats"
+         title="{% trans 'Expand details' %}"></i>
+    </a>
+    <div class="path-summary-collapsed">
+      {% top_scorers project=project.code language=language.code limit=3 format="as_ul"%}
     </div>
-    <div id="js-stats-checks" class="hide stats-checks">
-      <h3>{% trans "Failing Checks" %}</h3>
-      <div class="bd">
-      {% for item in check_categories %}
-        <div id="check-{{ item.name }}" class="checks js-checks">
-          <h4 class="clear">{{ item.title }}</h4>
-          {% for check in item.checks %}
-          <div class="check js-check" data-code="{{ check.code }}">
-            {% if can_translate_stats %}
-            <div class="check-name"><a class="check-data" href="{{ check.url }}">{{ check.title }}</a></div>
-            <div class="check-count"><a class="check-data" href="{{ check.url }}">0</a></div>
-            {% else %}
-            <div class="check-name"><span class="check-data">{{ check.title }}</span></div>
-            <div class="check-count"><span class="check-data">0</span></div>
-            {% endif %}
+    <div class="path-summary-expanded">
+      <div class="summary-1-col">
+        <h3 class="top">{% trans "Translation Statistics" %}</h3>
+        <div class="bd">
+          <table class="stats" lang="{{ LANGUAGE_CODE }}" dir="{% locale_dir %}">
+            <tbody>
+              {% for item in translation_states %}
+              <tr id="stats-{{ item.state }}">
+                <td id="stats-name">{{ item.title }}</td>
+                <td class="stats-number">
+                  {% if can_translate_stats %}
+                  <a href="{{ item.url }}" class="stats-data"></a>
+                  {% else %}
+                  <span class="stats-data"></span>
+                  {% endif %}
+                </td>
+                <td class="stats-percentage"><span></span>%</td>
+              </tr>
+              {% endfor %}
+            </tbody>
+          </table>
+        </div>
+        <div id="js-stats-checks" class="hide stats-checks">
+          <h3>{% trans "Failing Checks" %}</h3>
+          <div class="bd">
+            <table class="sortable checks">
+              <thead>
+                <tr>
+                  <th id="js-checkname-th" class="sorttable_customkey"></th>
```

Comment:
With better alternatives I meant that _doing the sorting directly on top of the data_ is way better (and saner) as opposed to doing it in the DOM.